docs/kit/operation-view/installation_guide.md

```diff
@@ -0,0 +1,116 @@
+Installation Guide for Tractus-X EDC
+==================
+This guide describes how to install the [Tractus-X EDC](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/charts/tractusx-connector) via helm chart with Hashicorp Vault + PostgreSQL.
+
+Prerequisite
```

Comment:
I think it is enough to state that DAPS, Postgres and Hashicorp must be available and accessible beforehand, and access vectors (database name, passwords, tokens, etc.) must be known. This is an installation guide, not a sample, so we can safely assume users have already worked through the sample, and/or have a running environment. 
For example, I would only state that "a valid DAPS certificate and client-id must exist" and link to wherever that process is defined.

At most, I would explain configuration that is specific to the connector, such as the use of a static vault token. 

This will make the install guide much more succinct and quicker to read.

Reply:
Is DAPS a prereq at this point since it is going away? The prereq could be "an identity provider"